File: week2/Task-JS/task4.js
function getNumber(index){
    const term = -5 * Math.floor(index / 3);
    const number = index * 4 + term

    return console.log(number)
    }

getNumber(1); // print 4
getNumber(5); // print 15
getNumber(10); // print 25
getNumber(30); // print 70

File: week2/Task-Py/task4.py
def get_number(index):
    term = -5 * (index // 3)
    number = index * 4 + term
    print(number) 

get_number(1) # print 4
get_number(5) # print 15
get_number(10) # print 25
get_number(30) # print 70